chromium_src/ios/chrome/browser/ui/webui/about_ui.cc

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/webui/about_ui.h"  // IWYU pragma: export
+#include "ios/chrome/browser/shared/model/url/chrome_url_constants.h"  // IWYU pragma: export
+
+#define kChromeUIChromeURLsHost kChromeUIChromeURLsHost) {    \
```

Comment:
cc @mkarolin this is not undefined below, but the presubmit check is fine with that. a bug or some edge case?